qa/workunits/mon/crush_ops.sh

```diff
@@ -21,8 +21,12 @@ ceph osd crush rule create-simple bar default host
 
 # make sure we're at luminous+ before using crush device classes
 ceph osd require-osd-release luminous
-ceph osd crush set-device-class ssd osd.0
-ceph osd crush set-device-class hdd osd.1
+o0=`ceph osd create`
+ceph osd crush set-device-class ssd $o0
+expect_false ceph osd crush set-device-class hdd $o0
+ceph osd crush rm-device-class $o0
```

Comment:
Actually this will not work because `rm-device-class` will automatically recycle empty class..